Library/Formula/antlr.rb

```diff
@@ -13,4 +11,23 @@ def install
       java -classpath #{prefix}/antlr-#{version}-complete.jar:. org.antlr.v4.runtime.misc.TestRig "$@"
     EOS
   end
+
+  test do
+    path = testpath/"Expr.g4"
+    path.write <<-EOS.undent
+    grammar Expr;
+    prog:\t(expr NEWLINE)* ;
+    expr:\texpr ('*'|'/') expr
+        |\texpr ('+'|'-') expr
+        |\tINT
+        |\t'(' expr ')'
+        ;
+    NEWLINE :\t[\\r\\n]+ ;
+    INT     :\t[0-9]+ ;
+    EOS
+    ENV["CLASSPATH"] = ".:#{prefix}/antlr-#{version}-complete.jar:#{ENV["CLASSPATH"]}"
```

Comment:
@jacknagel Right, I edited my comment while you were writing yours. Do you want me to keep the `ENV.prepend` thing, or submit another pull-request to check if the argument is a directory **or a JAR** and use `ENV.prepend_path` here?